tracking/include/pcl/tracking/tracking.h

```diff
@@ -42,89 +42,87 @@
-)
-POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYR, pcl::tracking::_ParticleXYR)
-
-POINT_CLOUD_REGISTER_POINT_STRUCT (pcl::tracking::_ParticleXYZR,
-    (float, x, x)
-    (float, y, y)
-    (float, z, z)
-    (float, roll, roll)
-    (float, pitch, pitch)
-    (float, yaw, yaw)
-)
-POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYZR, pcl::tracking::_ParticleXYZR)
+// clang-format off
+POINT_CLOUD_REGISTER_POINT_STRUCT(pcl::tracking::_ParticleXYZRPY,
+                                  (float, x, x)
+                                  (float, y, y)
+                                  (float, z, z)
+                                  (float, roll, roll)
+                                  (float, pitch, pitch)
+                                  (float, yaw, yaw))
+POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYZRPY,
+                                   pcl::tracking::_ParticleXYZRPY)
+
+POINT_CLOUD_REGISTER_POINT_STRUCT(pcl::tracking::_ParticleXYRPY,
+                                  (float, x, x)
+                                  (float, y, y)
+                                  (float, z, z)
+                                  (float, roll, roll)
+                                  (float, pitch, pitch)
+                                  (float, yaw, yaw))
+POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYRPY,
+                                   pcl::tracking::_ParticleXYRPY)
+
+POINT_CLOUD_REGISTER_POINT_STRUCT(pcl::tracking::_ParticleXYRP,
+                                  (float, x, x)
+                                  (float, y, y)
+                                  (float, z, z)
+                                  (float, roll, roll)
+                                  (float, pitch, pitch)
+                                  (float, yaw, yaw))
+POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYRP,
+                                   pcl::tracking::_ParticleXYRP)
+
+POINT_CLOUD_REGISTER_POINT_STRUCT(pcl::tracking::_ParticleXYR,
+                                  (float, x, x)
+                                  (float, y, y)
+                                  (float, z, z)
+                                  (float, roll, roll)
+                                  (float, pitch, pitch)
+                                  (float, yaw, yaw))
+POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYR,
+                                   pcl::tracking::_ParticleXYR)
+
+POINT_CLOUD_REGISTER_POINT_STRUCT(pcl::tracking::_ParticleXYZR,
+                                  (float, x, x)
+                                  (float, y, y)
+                                  (float, z, z)
+                                  (float, roll, roll)
+                                  (float, pitch, pitch)
+                                  (float, yaw, yaw))
+POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::tracking::ParticleXYZR,
+                                   pcl::tracking::_ParticleXYZR)
+// clang-format on
```

Comment:
@kunaltyagi 
Had a doubt. Add namespace here too?

Also, I removed `pcl::tracking::` while adding namespaces, but not `pcl::`. I should do that too, right?

Reply:
> Add namespace here too?

No.

> I should do that too, right?

No. Remove only the `pcl::tracking::` at the definition level, not inside the definition/declarations please. Make the change as minimal as possible. What you're suggesting is something to be done in a separate PR.

Reply:
> Add namespace here too?
> 
> > No.

What is the reason for this?

Reply:
Macro didn't like being namespaced last time